### PR TITLE
Added German localization

### PR DESCRIPTION
**Pull request by drlellinger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2707,7 +2707,6 @@ Add a menuitem to the help menu for specifying support information.
 ```
 Software\Policies\Mozilla\Firefox\SupportMenu\Title = "Support Menu"
 Software\Policies\Mozilla\Firefox\SupportMenu\URL = "http://example.com/support"
-Software\Policies\Mozilla\Firefox\SupportMenu\Title = "S"
```
Comment (mueller-ma) [inline README.md]: I guess this should be
```suggestion
Software\Policies\Mozilla\Firefox\SupportMenu\AccessKey= "S"
```

File: windows/de-DE/firefox.adml [hunk 2]
```diff
@@ -495,10 +495,10 @@ Wenn Sie die Richtlinieneinstellung aktivieren, privates Surfen ist sowohl im Br
       <string id="RequestedLocales_Explain">If this policy is enabled, you can specify a list of requested locales for the application in order of preference. It will cause the corresponding language pack to become active.
 
 If this policy is disabled or not configured, the application will use the default locale.</string>
-      <string id="SecurityDevices">Security Devices</string>
-      <string id="SecurityDevices_Explain">If this policy is enabled, you can specify a list of PKCS #11 modules to be installed. Modules are specified as a name and a fully qualified path.
+      <string id="SecurityDevices">Sicherheitsgeräte</string>
+      <string id="SecurityDevices_Explain">Wenn diese Richtlinie aktiviert ist, können Sie eine Liste mit PKCS #11 Modulen definieren. Module werden mit einem Namen und einem qualifizierten Pfad benannt.
```
Comment (mueller-ma) [inline windows/de-DE/firefox.adml]:
```suggestion
      <string id="SecurityDevices_Explain">Wenn diese Richtlinie aktiviert ist, können Sie eine Liste mit PKCS #11 Modulen definieren. Module werden mit einem Namen und einem vollständigen Pfad benannt.
```

**Comment by mkaply** [comment]
Can you remove the README change and address the other suggestion? I'll fix that in a separate pull.

Thanks!

**Comment by drlellinger** [comment]
Pull Request is now conflict free

**Comment by mkaply** [comment]
Thanks!

**Pull request merged**